typedb/connection/session.py

```diff
@@ -18,13 +18,10 @@
 # specific language governing permissions and limitations
```

Comment:
We've moved the pulse scheduling logic out of this `Session` class and into `Client`.